### PR TITLE
skeleton for know your customer screens

### PR DESCRIPTION
**Pull request by jpangelle**
### Description

This pull request implements the basic KYC components of the onboarding flow including placeholders for the ShuftiPro iframes.

### Approvals

- [x] Dev

### CODE REVIEW

File: fabric/src/components/Stepper/index.tsx [hunk 1]
```diff
@@ -187,7 +187,6 @@ export const Stepper = (props: StepperProps) => {
   }, [props.activeStep])
 
   const stepItems = steps.map((step, index) => {
-    console.log({ step })
```
Comment (jpangelle) [inline fabric/src/components/Stepper/index.tsx:190]: whoops

File: centrifuge-app/src/components/DebugFlags/context.ts [hunk 2]
```diff
@@ -2,12 +2,12 @@ import * as React from 'react'
 import { debug, flagsConfig, Key } from './config'
```
Comment (jpangelle) [inline centrifuge-app/src/components/DebugFlags/context.ts:2]: some random `prettier` formatting fix

File: centrifuge-app/src/pages/Onboarding/KnowYourCustomer/AuthorizedSignerVerification.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  })
+
+  const {
+    mutate: verifyAuthorizeSigner,
+    isLoading,
+    isError,
+  } = useMutation(
+    async () => {
+      await new Promise((resolve) => setTimeout(resolve, 3000))
+      // const response = await fetch(`${import.meta.env.REACT_APP_ONBOARDING_API_URL}/authorizedSignerVerification`, {
```
Comment (jpangelle) [inline centrifuge-app/src/pages/Onboarding/KnowYourCustomer/AuthorizedSignerVerification.tsx:60]: eventually the kyc endpoint will go here

File: centrifuge-app/src/pages/Onboarding/KnowYourCustomer/AuthorizedSignerVerification.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+export const AuthorizedSignerVerification = ({ nextKnowYourCustomerStep }: Props) => {
+  const { authToken } = useAuth()
+  const [isAccurate, setIsAccurate] = useState(false)
+
+  const formik = useFormik({
+    initialValues: {
+      name: '',
+      dateOfBirth: '',
+      countryOfCitizenship: '',
```
Comment (sophialittlejohn) [inline centrifuge-app/src/pages/Onboarding/KnowYourCustomer/AuthorizedSignerVerification.tsx:44]: curious why isAccurate is not part of the formik data?

Comment (jpangelle) [reply centrifuge-app/src/pages/Onboarding/KnowYourCustomer/AuthorizedSignerVerification.tsx:43]: resolved

**Review by sophialittlejohn — APPROVED** [review]
💯

**Pull request merged**